File: week1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float cash;
    int cnt = 0;
    do
    {
        cash = get_float("Change owed: ");
    }
    while (cash < 0.0099);
    cash = round(cash * 100);
    while (cash >= 25)
    {
        cnt++;
        cash -= 25;
    }
    while (cash >= 10)
    {
        cnt++;
        cash -= 10;
    }
    while (cash >= 5)
    {
        cnt++;
        cash -= 5;
    }
    while (cash > 0)
    {
        cnt++;
        cash -= 1;
    }
    printf("%d\n", cnt);
    return (0);
}

File: week6/cash/cash.py
from cs50 import get_float

cash = 0
cnt = 0
while cash < 0.0099:
    cash = get_float("Change owed: ")
cash = int(cash * 100)
while cash >= 25:
    cnt += 1
    cash -= 25
while cash >= 10:
    cnt += 1
    cash -= 10
while cash >= 5:
    cnt += 1
    cash -= 5
while cash > 0:
    cnt += 1
    cash -= 1
print(cnt)
